client/landing/jetpack-cloud/routes.js

```diff
@@ -14,8 +14,11 @@ import {
 	makeLayout,
 	setupSidebar,
 } from './controller';
+import { normalize } from 'lib/route';
 
 const router = () => {
+	page( '*', normalize );
+
```

Comment:
Also would be cool to create empty routes for these new pages that we're adding in the menu. I just don't favor introducing links to 404s. Or maybe we could just link those menu links that don't have corresponding pages to just `#`?

Reply:
Sure thing! I'm going to probably use `#` for now since the IA is still in flux and we don't really know what are we going to need.